charts/addons/requirements.yaml

```diff
@@ -8,22 +8,22 @@ dependencies:
   repository: https://kubernetes-charts-incubator.storage.googleapis.com/
   condition: aws-alb-ingress-controller.enabled
 - name: elasticsearch
-  version: "1.15.1"
-  repository: https://kubernetes-charts.storage.googleapis.com/
+  version: "7.8.1"
+  repository: https://helm.elastic.co/
   condition: elasticsearch.enabled
 - name: fluentd-elasticsearch
-  version: "1.5.0"
-  repository: https://kubernetes-charts.storage.googleapis.com/
+  version: "9.6.0"
+  repository: https://kiwigrid.github.io/
   condition: fluentd-elasticsearch.enabled
 - name: kibana
-  version: "1.1.0"
-  repository: https://kubernetes-charts.storage.googleapis.com/
+  version: "7.8.1"
+  repository: https://helm.elastic.co/
   condition: kibana.enabled
 - name: metrics-server
   version: "2.8.4"
   repository: https://kubernetes-charts.storage.googleapis.com/
   condition: metrics-server.enabled
 - name: ingress-azure
-  version: "1.0.0"
+  version: "1.2.0-rc3"
```

Comment:
Why `1.2.0-rc3`? Isn't `1.2.0` more current?

Reply:
Yes. This will be updated to 1.2.0.